src/components/UserInfo/UserInfo.jsx

```diff
@@ -1,4 +1,11 @@
-// Add the required props
-export const UserInfo = () => (
-  <>UserInfo markup</>
-);
+import './UserInfo.scss';
+
+export const UserInfo = ({ user }) => {
+  const { email, username } = user;
+
+  return (
+    <a className="UserInfo" href={email}>
```

Comment:
Using the email as href for anchor tag is not a recommended approach. Instead of href={email}, consider using 'mailto:' prefix to make it clickable and open user's default email client. For example, 'href={`mailto:${email}`}.'